### PR TITLE
Update guest-access-PowerShell.md

### PR DESCRIPTION
**Pull request by pupara**
Added Notes
Replaced SkypeOnlineConnector with MicrosoftTeams

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/guest-access-PowerShell.md [hunk 1]
```diff
@@ -43,8 +43,12 @@ If you want to block guests in Teams and still want to allow them to access Shar
  
 2.  Connect a PowerShell session to the Skype for Business Online endpoint.
 
+> [!NOTE]
+> Skype for Business Online Connector is currently part of latest Teams PowerShell Module.
```
Comment (JohanFreelancer9) [inline Teams/guest-access-PowerShell.md:47]:
```suggestion
> The Skype for Business Online Connector is currently part of the latest Teams PowerShell module.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by scanum** [comment]
@LanaChin PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**